sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobRange.java

```diff
@@ -128,24 +128,26 @@ static EncryptedBlobRange getEncryptedBlobRangeFromHeader(String stringRange, En
             case ENCRYPTION_PROTOCOL_V2:
                 // Calculate offsetAdjustment.
                 // Get the start of the encryption region for the original offset
-                long regionNumber = originalRange.getOffset() / GCM_ENCRYPTION_REGION_LENGTH;
+                // grab the region length from the metadata
```

Comment:
Let's remove this comment as it sort of interrupts the two comments just above it.